staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -778,6 +778,10 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	RegisterNode *bool `json:"registerNode,omitempty"`
+	// Tracing specifies the options of tracing.
```

Comment:
Same comments as on the internal type -- can we get a bit more info here, as well as what the defaults are?